docs/debugging-s2i.md

```diff
@@ -0,0 +1,116 @@
+
+Troubleshooting
+=================
+
+This document contains some tips and suggestions for troubleshooting the use of source-to-image ("s2i") to inject source into Docker images.
+
+Can't Find The `sti` Executable
+--------------
+
+Whether by  running `make build`,  running `hack/build-release.sh`, or downloading one of our official releases,  look under the `_output/local/go/bin` directory.
+
+(GGM NOTE:  stems from https://github.com/openshift/source-to-image/issues/74 )
+
+Image Mechanics
+--------------
+
+By default, s2i will use the local builder image if available on the host you are running the `sti` command from, instead of pulling it from the remote docker registry.  If that image becomes stale and out of date, either use the `--forcePull=true` option when running the `sti` command,
```

Comment:
s/forcePull/force-pull
Besides this is the default behaviour, you can turn it off if you want, otherwise we always reach out to get the latest builder image (see [here](https://github.com/openshift/source-to-image/blob/master/cmd/sti/main.go#L183)).